tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+skipsdist = True
```

Comment:
It might make sense to have a list of environments here as well. In CI, each job has its env defined from the matrix but what should happen when I run ¯tox¯ command locally? I'd expect it to run tests in all supported environments which should be listed here.

Reply:
Resolved by https://github.com/sclorg/container-ci-suite/pull/8/commits/701c1550cf1081abbaebbd54da03528453336851